app/presenters/tree_builder_miq_action_category.rb

```diff
@@ -0,0 +1,53 @@
+class TreeBuilderMiqActionCategory < TreeBuilder
+  has_kids_for Classification, [:x_get_tree_classification_kids]
+
+  private
+
+  def override(node, object, _pid, _options)
+    img = "blank.gif"
+    leaf = !object.entries.any?
```

Comment:
`leaf = object.parent_id != 0` doesn't trigger a DB query and *probably* does the same thing.

(Well, there is a difference for objects with non-null parent_id which do have entries, but that doesn't seem to happen.)